src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.scss

```diff
@@ -0,0 +1,5 @@
+@use '@carbon/layout/scss/spacing';
+
+:host ::ng-deep .cds--popover--open > .cds--popover > .cds--popover-content {
+  margin-left: spacing.$spacing-09;
+}
```

Comment:
why is this? If its for fixing the tooltip getting hidden behind some panels, then the ideal way is to use what carbon already provides. I just looked [into the docs](https://angular.carbondesignsystem.com/?path=/docs/components-tooltip--docs) and saw they have an `autoAlign` feature which takes care of this. So all you need to do is to add this input to the `helper.component.html` cds-tooltip component.

Reply:
i will check

Reply:
removed this css code